### DISCUSSION TITLE
Select loadingMessage props

### DISCUSSION THREAD

**Discussion by maxencetasse** [BearStudio/start-ui-web#251] [Q&A] [answered]
Hi,
I have a question about loadingMessage props from `<Select>` component. 

When I use that props with String return like that :
<Select
     loadingMessage={() => {
        return 'Chargement...';
      }}
      noOptionsMessage={() => {
        return 'Pas de résultat...';
      }}
    />
noOptionsMessage works but loadingMessage don't work.
When I use it with component, nothing appears on screen :
<Select
     loadingMessage={() => {
        return `<Text>Chargement...</Text>;`
      }}
      noOptionsMessage={() => {
        return 'Pas de résultat...';
      }}
    />
How can I fix this problem please ?
Thank you

**Comment by ivan-dalmet** [comment]
Hi @maxencetasse there is a little bug here. I will fix it :)

**Comment by ivan-dalmet** [comment, marked as answer]
@maxencetasse You can see the fix here :) https://github.com/BearStudio/start-ui-web/pull/252/files

**Reply by maxencetasse** [reply]
Thank you ! @ivan-dalmet
